go/grpcwebproxy/main.go

```diff
@@ -32,8 +34,8 @@ var (
 	flagHttpPort    = pflag.Int("server_http_debug_port", 8080, "TCP port to listen on for HTTP1.1 debug calls.")
 	flagHttpTlsPort = pflag.Int("server_http_tls_port", 8443, "TCP port to listen on for HTTPS (gRPC, gRPC-Web).")
 
-	flagAllowAllOrigins = pflag.Bool("allow_all_origins", false, "allow requests from any origin.")
-	flagAllowedOrigins  = pflag.StringSlice("allowed_origins", nil, "comma-separated list of origin URLs which are allowed to make cross-origin requests.")
+	flagAllowedOrigins = pflag.StringSlice("allowed_origins", nil, "comma-separated list of origin URLs which are allowed to make cross-origin requests, defaults to denying all cross origin requests, whitelist all origins with '*'")
```

Comment:
We might want to keep the `flagAllowAllOrigins` as well and remove it later to make this transition less painful for users.